Source/WebCore/platform/graphics/SourceBrush.cpp

```diff
@@ -36,14 +36,14 @@ SourceBrush::SourceBrush(const Color& color, std::optional<Brush>&& brush)
 
 const AffineTransform& SourceBrush::gradientSpaceTransform() const
 {
-    static NeverDestroyed<AffineTransform> identity;
+    static constexpr AffineTransform identity;
```

Comment:
I wouldn't be surprised if the compiler de-duplicated this and the one in `RenderObject::localToParentTransform`. But if you want to avoid having this line in both functions, you could add that

```
static constexpr AffineTransform identity;
```

as a public member variable of `AffineTransform`, then do `return AffineTransform::identity;` here and in `RenderObject::localToParentTransform`. (In C++17, a static constexpr data member is implicitly an "inline variable", which means you don't need to write the variable definition in the cpp file, and the compiler will merge all of the instances of the variable across translation units.)

Reply:
That's a good idea, but I can't put it inside `AffineTransform` as a member because of https://stackoverflow.com/questions/36863503/struct-is-non-literal-type/36863691#36863691. I'll put it in `WebCore` next to the `AffineTransform` class.

Reply:
That's annoying. I guess you'll need to mark them explicitly as `inline` since they'll no longer be static member variables.